nomic/dataset.py

```diff
@@ -656,87 +687,86 @@ def _get_atoms(self, ids: List[str]) -> List[Dict]:
         else:
             raise Exception(response.text)
 
-
-class AtlasProject(AtlasClass):
+class AtlasDataset(AtlasClass):
     def __init__(
         self,
-        name: Optional[str] = None,
+        identifier: Optional[str] = None,
         description: Optional[str] = 'A description for your map.',
         unique_id_field: Optional[str] = None,
         modality: Optional[str] = None,
-        organization_name: Optional[str] = None,
         is_public: bool = True,
         project_id=None,
         reset_project_if_exists=False,
         add_datums_if_exists=True,
+        organization_name=None
     ):
         """
-        Creates or loads an Atlas project.
-        Atlas projects store data (text, embeddings, etc) that you can organize by building indices.
-        If the organization already contains a project with this name, it will be returned instead.
+        Creates or loads an AtlasDataset.
+        AtlasDataset's store data (text, embeddings, etc) that you can organize by building indices.
+        If the organization already contains a dataset with this name, it will be returned instead.
 
         **Parameters:**
 
-        * **project_name** - The name of the project.
+        * **identifier** - The dataset identifier (format is `organization/project`) or just dataset and we will auto-infer from the default org.
         * **description** - A description for the project.
```

Comment:
It's weird to allow passing the *description* here but not the project name that appears on the top. We could allow passing the display name as an argument, but by default use the slug and not allow fetching by display name.